application/commands/ImportSurveyCommand.php

```diff
@@ -46,14 +54,25 @@ public function run($sArgument)
         switch ($command) {
             case "import-file": {
                 if (count($sArgument) < 2) {
-                    throw new Exception("You need to specify the file to import from");
+                    throw new Exception("You need to specify the import path");
                 }
-                $filename = $sArgument[1];
-                if (!preg_match('/^[a-zA-Z0-9_\.]*$/', $filename)) {
+                $fullPath = $sArgument[1];
+
+                if (($lastSlashPosition = strripos($fullPath, DIRECTORY_SEPARATOR)) === false) {
+                    throw new Exception("Invalid path");
+                }
+                $path = substr($fullPath, 0, $lastSlashPosition);
+                $filename = substr($fullPath, $lastSlashPosition + 1);
+                if (!preg_match('/^[a-zA-Z0-9_\. ]*$/', $filename)) {
                     throw new Exception("Your filename can only contain letters, digits and dot");
                 }
                 $furtherParams = $sArgument[2] ?? null;
-                $this->importFile($filename, $furtherParams);
+                $result = $this->importFile($path, $filename, $furtherParams);
+                if (is_array($result) && isset($result['newsid'])) {
+                    echo $result['newsid'];
+                } else {
+                    echo "something went wrong";
```

Comment:
Bit imprecise :) Also should use gT for translation.

Reply:
Applied the change.

Reply:
Where? ^^ No push?

Reply:
Forgot to push it. It's pushed now. Excuse me for the inconvenience